src/domain/SignupConfig.ts

```diff
@@ -0,0 +1,40 @@
+import { ISignUpConfig } from "aws-amplify-react/lib-esm/Auth/SignUp";
+
+const SignUpConfig: ISignUpConfig = {
+  hideAllDefaults: true,
+  signUpFields: [
+    {
+      label: 'Email',
```

Comment:
Will they also choose a username or will their email be the username?

Reply:
Their email will be their username